language/fr/pages_common.php

```diff
@@ -39,5 +39,5 @@
 //
 
 $lang = array_merge($lang, array(
-	'PAGES_VIEWONLINE'		=> 'Consulte %s',
+	'PAGES_VIEWONLINE'		=> 'Consulte la page “%s”',
```

Comment:
This was correct as it was. It should only say viewing, and leave the rest of the name as variable. It also should not be wrapping the page name in quotes.